src/index.js

```diff
@@ -137,26 +139,7 @@ function getSlotNameFromNode(node) {
 }
 
 function slotNodeIntoSlot(slot, node, insertBefore) {
-  // Don't slot nodes that have content but are only whitespace. This is an
```

Comment:
Unfortunately both v0 and v1 allow this, so I've relaxed this check to allow whitespace of all sorts. This just means that if we want fallback content to display that we need to ensure there's no unnecessary whitespace in our DOM.